duo_client/admin.py

```diff
@@ -3282,6 +3282,17 @@ def __init__(self, account_id, **kwargs):
         """Initializes an AccountAdmin for administering a child account.
            account_id is the account id of the child account.
            See the Client base class for other parameters."""
+        child_api_host =  Accounts.child_map.get(account_id, None)
+        if child_api_host is None:
+            child_api_host = kwargs.get('host')
+            try:
+                accounts_api = Accounts(**kwargs)
+                accounts_api.get_child_accounts()
+                child_api_host =  Accounts.child_map.get(account_id, kwargs['host'])
+            except RuntimeError:
+                pass
+        kwargs['host'] = child_api_host
```

Comment:
if a RuntimeError does indeed happen, you might set kwargs['host'] to None.  will that cause a problem?

Reply:
Technically, it would cause a problem with the successful API call. However, in this scenario, the Runtime error should only occur if there was bad data being specified in the first place, which would still result in an error.  It's an error either way.

Reply:
works for me